es/ch08.md

```diff
@@ -0,0 +1,11 @@
+# Acerca de nodos remoto
+
+Los nodos remotos pueden ser muy útiles si no eres capaz o no deseas descargar la cadena de bloques completa, pero hemos advertido que nodos remotos maliciosos pueden comprometer el nivel de privacidad de tu transacción, rastreando tu dirección IP, o, en casos extremos, mostrando incluso la cantidad de la transacción. Dicho esto, una lista de nodos remotos puede encontrarse en [moneroworld.com](https://moneroworld.com). Mantén en mente que cualquiera es capaz de agregar un nodo a esa lista, no debes considerar esos nodos como seguros o confiables, siempre ejecuta tu propio nodo para la mejor privacidad.
+
+## Nodos Bootstrap
+
+Un nodo bootstrap es un nodo remoto para usar mientras también se sincroniza la cadena de bloques. Esto difiere a un nodo remoto en que usar un nodo remoto no sincronizará la cadena de bloques a un nodo local.    
+Si se utiliza la característica de nodo bootstrap automáticamente se cambiará al nodo local cuando la cadena de bloques se sincronice totalmente, independientemente de si el usuario eligió el nodo remoto por él mismo o el nodo remoto se especificó por defecto.    
+Más información en https://getmonero.org/resources/moneropedia/bootstrap-node.html.
```

Comment:
You could link to the spanish moneropedia here

Reply:
Got it (Y)